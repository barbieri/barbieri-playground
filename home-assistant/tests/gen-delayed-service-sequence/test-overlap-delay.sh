#!/bin/sh

gen-delayed-service-sequence.py --overlap-steps=5s -- \
    "00:01:00=switch.light1" \
    "1m=switch.light2" \
    "60s=switch.light3"
