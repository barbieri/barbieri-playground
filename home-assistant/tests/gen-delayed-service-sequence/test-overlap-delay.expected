alias: Sequence (2m 10s)
mode: single
sequence:
- service: switch.turn_on
  target:
    entity_id: switch.light1
- delay: 0:01:00
- service: switch.turn_on
  target:
    entity_id: switch.light2
- delay: 0:00:05
- service: switch.turn_off
  target:
    entity_id: switch.light1
- delay: 0:01:00
- service: switch.turn_on
  target:
    entity_id: switch.light3
- delay: 0:00:05
- service: switch.turn_off
  target:
    entity_id: switch.light2
- delay: 0:00:00
- service: switch.turn_off
  target:
    entity_id: switch.light3
